carts/carts.html

```diff
@@ -0,0 +1,165 @@
+
+        <!-- ******************** Navbar right started******************** -->
+
+        <div class="right-navbar">
+          <ul class="nav-icons list-style-none">
+            <li>
+              <button href="/" class="flex-col nav-icon-link">
+                <span class="badge-icon">
+                  <i class="fas fa-user icon-size"></i>
+                </span>
+                <span class="nav-list-link">Login</span>
+              </button>
+            </li>
+            <li>
+              <button href="/" class="flex-col nav-icon-link wishlist-button">
+                <span class="badge-icon">
+                  <i class="fas fa-heart"></i>
+                  <span class="badge-count">10</span>
+                </span>
+                <span class="nav-list-link">Wishlist</span>
+              </button>
+            </li>
+            <li>
+              <button href="/" class="flex-col nav-icon-link">
+                <span class="badge-icon">
+                  <i class="fas fa-shopping-cart"></i>
+                  <span class="badge-count">10</span>
+                </span>
+                <span class="nav-list-link">Cart</span>
+              </button>
+            </li>
+          </ul>
+        </div>
+        <!-- ******************navbar right ended**************** -->
+      </nav>
+
+      <main>
+        <div class="space-1rem"></div>
+        <h2 class="cart-heading text-sm mt-2">MY CART</h2>
+        <div class="space-3rem"></div>
+        <div class="width-800">
+          <div class="col-60-pc">
+            <div class="card-horizontal">
+              <div class="card-horizontal-image-container">
+                <img
+                  class="card-image-horizontal"
+                  src="https://cdn.shopify.com/s/files/1/0550/0465/9885/products/shoe-15.jpg?v=1615348374"
+                  alt="shoes"
+                />
+              </div>
+
+              <div class="card-horizontal-text-container">
+                <h2 class="card-horizontal-heading text-sm">Mufti</h2>
+                <p class="font-semibold card-horizontal-sub-heading text-xs my-2">
+                  Boys T-shirt
+                </p>
+                <p class="card-horizontal-product-desc text-xs">
+                  Black solid casual shirt, has a spread collar, button placket,
+                </p>
+                <div class="card-quantity flex-row">
+                  <button class="floating-action-btn">
+                    <i class="fas fa-plus"></i>
+                  </button>
+                  <p class="text-xs">3</p>
+                  <button class="floating-action-btn">
+                    <i class="fas fa-minus"></i>
+                  </button>
+                </div>
+                <span class="discount-price font-semibold">Rs.1500 </span>
+                <span class="original-price"> Rs.2200</span>
+                <small class="discount-rate">(15% OFF)</small>
```

Comment:
span is not a semantic element, so you can replace span with p

Reply:
changed it p tag !!